docs/getting-started/integrating-codacy-with-your-git-workflow.md

```diff
@@ -0,0 +1,59 @@
+
+Once you've [configured your repository](configuring-your-repository.md) to best match your use case, integrate Codacy with your Git workflow to display analysis results and code coverage as status checks on your pull requests.
+
+In particular, you can configure quality gates to block merging pull requests that don't meet the quality standards of your team. This ensures the quality of the changes to your codebase, preventing the introduction of security issues and untested code.
+
+To integrate Codacy with your Git workflow, follow these steps:
+
+1.  [Configuring the quality gate rules](#configuring-gate)
+1.  [Activating the Git provider integration](#git-provider-integration)
+1.  [Blocking merging pull requests](#blocking-pull-requests)
+
+## 1. Configuring the quality gate rules {: id="configuring-gate"}
+
+[Review and adjust the quality settings](../repositories-configure/adjusting-quality-settings.md) of your repository to decide which pull requests should fail the Codacy quality gate.
+
+!!! tip
+    The default values of the quality gate rules are designed to help maintain the current code quality of your repository. In particular, the default value for the coverage rule might be demanding. Depending on factors such as the current code quality of your repository and the maturity of your team practices, consider the balance between implementing stricter quality gates and the possibility of delaying or blocking the development progress.
+
+    Codacy generally recommends that on a first stage you configure rules that focus on stopping new critical issues from entering your code base, such as:
+
+    -   High severity issues
+    -   Security issues
+    -   Considerable drops in code coverage
+
+!!! important
+    **If you want to use code coverage** to block merging pull requests that don't meet your standards, make sure that you enable the rule **Diff coverage is under** or **Coverage variation is under**. This is required for Codacy to report the coverage status directly on your pull requests.
+
+![Adjusting the quality settings](../repositories-configure/images/quality-settings.png)
+
+## 2. Activating the Git provider integration {: id="git-provider-integration"}
+
+Follow the instructions for [GitHub](../repositories-configure/integrations/github-integration.md#enabling), [GitLab](../repositories-configure/integrations/gitlab-integration.md#enabling), or [Bitbucket](../repositories-configure/integrations/bitbucket-integration.md#enabling) depending on your Git provider, and make sure that you:
+
+1.  Enable the Git provider integration
+1.  Enable the option **Status checks** (GitHub) or **Pull request status** (GitLab and Bitbucket)
+
+![Enabling your Git provider integration](../repositories-configure/integrations/images/github-integration.png)
+
+## 3. Blocking merging pull requests {: id="blocking-pull-requests"}
+
+<!--TODO Add discussion about when it's time for customers to actually block merging pull requests-->
```

Comment:
What are out main points here? Something close to [the admonition above](https://github.com/codacy/docs/pull/1584/files#diff-999addb93ee2b841e94ab56f79160505a69c497ce464c762f2ea347a22d6a481R18)?

I'm thinking:
* Ensure your setup works (we mention this already)
* Test it out for a while before you turn the switch and let it block you

Reply:
Indeed, the key information here is making sure that the user is "happy" with their configuration so that they don't inadvertently block the work of the development teams.

All of this is already conveyed in the `important` admonition below, so I would experiment with transforming the admonition into the intro. We can round it off a bit better with:

* An introductory sentence explaining what it means or why it can be a good idea to block merging pull requests
* Changing the "Finally, follow the instructions [...]" to something along the lines of "Once you've tested out Codacy for a while and you're happy with the level of feedback provided, follow the instructions [...]"

Reply:
Besides these improvements, now I'm definitely convinced that we must mark the section for blocking the merge of pull requests as optional since not all customers will want to do it - and indeed, most customers **won't be ready** to do it during the onboarding.